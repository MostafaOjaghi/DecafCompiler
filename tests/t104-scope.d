void main() {
	int a;
	a = 5;
	{
		int a;
		a = 10;
		Print(a);
	}
	Print(a);
	{
		a = 22;
		Print(a);
	}
	Print(a);
}
