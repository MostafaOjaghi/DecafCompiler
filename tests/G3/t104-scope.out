10
5
22
22
